cypress/component/rich-text/RichTextPage.ts

```diff
@@ -114,23 +102,11 @@ export class RichTextPage {
     cy.getRichTextField().should((field) => {
       expect(field.getValue()).to.deep.equal(expectedValue);
     });
-
-    // There can't be any validation error
-    this.expectNoValidationErrors();
-  }
-
-  expectNoValidationErrors() {
-    cy.editorEvents()
-      .then((events) => {
-        return events.filter((ev) => isValidationEvent(ev) && ev.value.length > 0);
-      })
-      .should('be.empty')
-      .as('validationErrors');
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   expectTrackingValue(expectedValue: any) {
-    getIframeWindow()
+    cy.window()
       .should((win: any) => {
         expect(win.actions).to.deep.equal(expectedValue);
```

Comment:
Are we still using the `window` object? I thought now that we can directly passed a mocked `onAction` we can get rid of this?

Reply:
Yeah, this is not used anymore, missed this one.